### PR TITLE
Ticket 3 Pinging the Logout Controller

### PR DESCRIPTION
**Pull request by samsoncsyu7777**
### What this PR does (required):
- Pinging the Logout Controller

### Screenshots / Videos (front-end only):
- Post at least one screenshot to show the feature in action. **Only required in front-end work**

### Any information needed to test this feature (required):
- npm run test

### Any issues with the current functionality (optional):
- List any problems where you were not able to complete all tasks on ticket
- List any problems this PR may cause for the project or other active PRs
- Post a screenshot/video of the problem, along with detailed console outputs where applicable

### CODE REVIEW

File: server/test/logout.spec.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+const app = require("../app.js");
+
+chai.should();
+chai.use(chaiHttp);
+
+describe("Pinging the Logout controller: GET /logout", () => {
+  it("it should return 200 and logout response", (done) => {
+    chai
+      .request(app.server)
+      .get(`/auth/logout`)
```
Comment (rajivtitus) [inline server/test/logout.spec.js]: Just so we are aware, GET isn't the verb that makes the most sense for a /logout path! (see this [article](https://www.baeldung.com/logout-get-vs-post) for more details on why) For now though I think this is fine to leave as-is 👍🏼

The existing implementation may have thrown you off but this is the pitfall of developers building tests - we have to be careful not to be biased by our knowledge of the implementation and focus on specs!

**Review by rajivtitus — APPROVED** [review]
Good work! Please address comment prior to merging. Thank you!

File: server/test/logout.spec.js (hunk 1 above)
Comment (samsoncsyu7777) [reply server/test/logout.spec.js]: I see. May I change the Logout Route from GET to POST?

Comment (rajivtitus) [reply server/test/logout.spec.js]: Yes, please do so if possible :)

Comment (samsoncsyu7777) [reply server/test/logout.spec.js]: Done

**Pull request merged**